hack/run-wsu-ci-e2e-test.sh

```diff
@@ -6,13 +6,9 @@ set -o pipefail
 WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
 TEST_DIR=$WMCO_ROOT/tools/ansible/tasks/wsu/test/e2e
 
-#TODO: Currently exiting early as the CI operator job which runs this does not have pip installed,
-#      which is a requirement to run this script
-exit 0
-
+sleep 1000
```

Comment:
Why do you need the sleep? Hint: adding a comment would have pre-empted me from asking this question :-)